### DISCUSSION TITLE
How can I migrate from Ollama to llama.cpp?

### DISCUSSION THREAD

**Discussion by iladshyan** [ggml-org/llama.cpp#9561] [Q&A]
I have previously installed multiple models using Ollama and now I want to migrate, how can I do that?

**Comment by marcingomulkiewicz** [comment]
On unix-like systems your models will be in ~/.ollama/models.blobs under funny names. If you want to know which is which, look inside ~/.ollama/models/manifests/registry.ollama.ai/library/* . Probably automatic copying/moving file could be done using jq and some bash-magic, alternatively you may simply symlink existing files to some more user friendly names.
